elastic/integrations
# Add AWS Health Integration

## Type of change

- Enhancement


## Proposed commit message

Create AWS Health Integration. 
AWS health metrics provide insights into the health of your AWS environment by monitoring various aspects such as open issues, scheduled maintenance events, etc.


## Checklist


## Author's Checklist



## How to test this PR locally

- elastic-package build
- elastic-package stack up -v -d --services package-registry
- Configure the AWS Health Integration.
- Verify the dashboard data
- Validate using Discover interface



## Related issues

- Relates  https://github.com/elastic/integrations/issues/8907


## Screenshots

<img width="1727" alt="Screenshot 2024-06-10 at 8 20 53 AM" src="https://github.com/elastic/integrations/assets/101976829/1e3e4f19-0e3b-4768-8272-3496bf7d61a5">
